CONTRIBUTING.md

```diff
@@ -16,9 +16,9 @@ examples by going through the repositories in
 
 ## Steps for Contributing
 
-1. Clone this repository for a basic template from which to implement your
-   example. Alternatively, clone one of our existing examples and modify that
-   instead.
+1. Follow GitHub's instructions to [create a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).
+   Alternatively, clone this repository (or one of our existing examples) for a basic
+   template from which to implement your example.
```

Comment:
```suggestion
   Alternatively, clone this repository (or one of our existing examples) as a 
   starting point from which to implement your example.
```

To prevent confusion with "template"